mycroft/skills/phillips_hue/vocab/en-us/ActivateSceneKeyword.voc

```diff
@@ -0,0 +1,6 @@
+activate scene
+activate seen
+set my lights to
+set scene
+set seen
```

Comment:
Since the STT engine aims to return one interpretation of each utterance, you shouldn't need to give these alternate spellings.